docs/delegation_design_spec/delegation_design_spec.tex

```diff
@@ -1207,6 +1207,13 @@ \subsubsection{Operational Key Certificates}
 They are included in the message (e.g.~block header) and the message
 itself is signed with \(sks_\text{hot}\).
 
+Operational keys will use a key evolving signature (KES) scheme.
+
+Operational key certificates will have a maximal lifetime after which they
+become invalid, to encourage pool operators to regularly rotate their
+operational key. The lifetime of the certificate will be chosen to be shorter
+than the time in which a KES key expires.
```

Comment:
Maybe worth noting that the ttl for an OKC is a system property, rather than a parameter in the certificate?